src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/SupplierAssessmentControllerTest.kt

```diff
@@ -42,10 +46,10 @@ class SupplierAssessmentControllerTest {
     whenever(userMapper.fromToken(token)).thenReturn(user)
     whenever(referralService.getSentReferralForUser(referral.id, user)).thenReturn(referral)
     whenever(supplierAssessmentService.getSupplierAssessmentById(any())).thenReturn(supplierAssessment)
-    whenever(supplierAssessmentService.createOrUpdateSupplierAssessmentAppointment(supplierAssessment, durationInMinutes, appointmentTime, user)).thenReturn(supplierAssessment.currentAppointment)
+    whenever(supplierAssessmentService.createOrUpdateSupplierAssessmentAppointment(supplierAssessment, durationInMinutes, appointmentTime, user, appointmentDeliveryType)).thenReturn(supplierAssessment.currentAppointment)
 
     val response = supplierAssessmentController.updateSupplierAssessmentAppointment(referral.id, update, token)
-
+    verify(appointmentValidator, times(1)).validateUpdateAppointment(any())
```

Comment:
_nit_ Is it significant how many times does this happen?

Reply:
It's not significant, but need verification that it happened at least once.

Reply:
It's not significant, I'll remove the times()